package.json

```diff
@@ -1914,6 +1914,7 @@
     },
     "dependencies": {
         "@enonic/fnv-plus": "^1.3.0",
+        "@fluentui/react": "^7.160.1",
```

Comment:
Are you actually using this? I thought you dropped it?

Reply:
I tried to use this wherever I could, so yes I am using this in places where it was easier to reuse + override styles than it was to write my own.

Reply:
yay for using fluentui/react.... its awesome..